Makefile

```diff
@@ -147,7 +147,11 @@ test: generate fmt vet envtest ## Run tests.
 
 .PHONY: e2etest
 e2etest: generate local-deploy chainsaw
-	GIT_REF=$(GIT_REF) $(CHAINSAW) test ./e2e
+	GIT_REF=$(GIT_REF) $(CHAINSAW) test ./e2e/default-capl-cluster --assert-timeout 600s --cluster child-cluster=/e2e/default-capl-cluster/child-cluster-kubeconfig.yaml
+
+.PHONY: e2etest-pr
+e2etest-quick: generate local-deploy chainsaw
+	GIT_REF=$(GIT_REF) $(CHAINSAW) test ./e2e --selector quick
```

Comment:
could we add something like an `E2E_SELECTOR` to the e2etest target and just specify something like  `E2E_SELECTOR=quick make e2etest` to reduce the code duplication here?

Reply:
Yup I can do that!

Reply:
if we have a blank selector here does that default to `all` or `none`? if it defaults to `all` we might be able to just do `-- selector ${E2E_SELECTOR}` on the e2etest target and just use env variables to control the selector

Reply:
So if we have a blank selector, it throws an error. Alternatively, we can pass in the entire flag to the `E2E_SELECTOR` which is a good option. But may we should rename it to `E2E_FLAGS`?

Also, another thing I found was that we can only pass one selector. When I passed in multiple selector labels, it was not able to pick up on the tests. Maybe we can ask them to add the feature to pass in multiple selector labels?

Reply:
yeah, I think E2E_FLAGS would work here